pyfar/dsp/filter.py

```diff
@@ -353,10 +420,10 @@ def bessel(signal, N, frequency, btype='lowpass', norm='phase',
         return signal_filt
 
 
-def peq(signal, center_frequency, gain, quality, peq_type='II',
-        quality_warp='cos', sampling_rate=None):
+def bell(signal, center_frequency, gain, quality, bell_type='II',
```

Comment:
I'd suggest a different name:
```suggestion
def parametric_eq(signal, center_frequency, gain, quality, bell_type='II',
```

Reply:
That would be fine for me. Let's wait for more votes and finalize the name during the next weekly.

Reply:
I'm not sure which name I'd prefer, as I think `parametric_eq` is more intuitive, but `bell` is more precise as one could see a high- and low-shelf as a kind of parametric eq, too. When forced to decide I'd probably go with `parametric_eq`.

Reply:
Following the comment from @al-de, I googled 'peq filter' to see how the term is generally used. Lots of the prominent results did not clearly distinguish between peq/bell and shelve filters and use peq as the general term for both (e.g. https://de.mathworks.com/help/audio/ug/parametric-equalizer-design.html).

Googling 'bell filter' shows that this term seems to be less ambiguous but maybe slightly less familiar. Other terms that you find in this search are 'bell eq' and 'peaking filter'.

I would still vote for bell, to avoid mixing peq/bell and shelve filters. It might also be more intuitive for a non-audio peer group, because bell well describes the filter shape.